src/aioquic/quic/congestion/base.py

```diff
@@ -1,5 +1,5 @@
 import abc
-from typing import Dict, Iterable, Optional, Protocol
+from typing import Dict, Iterable, Optional, Protocol, Any
```

Comment:
Please run the linters (black / ruff / mypy) locally to avoid CI runs,  this is guaranteed to not be OK as it's not sorted